### ISSUE TITLE
Does "Model.fit" already support named inputs?

### ISSUE THREAD

**Issue by nathannau** [SciSharp/TensorFlow.NET#779]
In tensorflow doc : (https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit)

Arguments "x" Can be : "A dict mapping input names to the corresponding array/tensors, if the model has named inputs."

But in tensorflow.net only two signature exist : 
    public void fit(IDatasetV2 dataset, IDatasetV2 validation_data = null, int batch_size = -1, int epochs = 1, int verbose = 1, float validation_split = 0, bool shuffle = true, int initial_epoch = 0, int max_queue_size = 10, int workers = 1, bool use_multiprocessing = false);
    public void fit(NDArray x, NDArray y, int batch_size = -1, int epochs = 1, int verbose = 1, float validation_split = 0, bool shuffle = true, int initial_epoch = 0, int max_queue_size = 10, int workers = 1, bool use_multiprocessing = false);
